server/util/src/test_helpers.rs

```diff
@@ -9,3 +9,13 @@ macro_rules! assert_matches {
         )
     }};
 }
+
+#[macro_export]
+macro_rules! assert_variant {
```

Comment:
this macro could be useful to others, i should have added it earlier for tests helpers